### PR TITLE
Append error msg to exception for test.assert_* method

### LINKED ISSUE #21988: Feature request for tf.test.TestCase.assertNotAllClose: do not log the differences
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: MacOs
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**: 1.10
- **Python version**: 3.6
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**:

### Describe the problem
When running this test, it logs the failure of `assertAllClose` which is not necessary.

### Source code / logs

    import tensorflow as tf
    test_case = tf.test.TestCase()
    test_case.assertNotAllClose([1, 2, 3], [2, 3, 4])

The logs are (I would expect no log):

    not close where =  (array([0, 1, 2]),)
    not close lhs =  [1 2 3]
    not close rhs =  [2 3 4]
    not close dif =  [1 1 1]
    not close tol =  [3.e-06 4.e-06 5.e-06]
    dtype = int64, shape = (3,)

### PR DESCRIPTION
**Pull request by facaiy**
Fix #21988.

### CODE REVIEW

**Comment by facaiy** [comment]
@yifeif Hi, yifei. Could you take a look? Thanks.

**Comment by facaiy** [comment]
ping @yifeif :-)

**Comment by facaiy** [comment]
Gentle ping @yifeif

File: tensorflow/python/framework/test_util.py [hunk 1]
```diff
@@ -1338,35 +1338,36 @@ def _assertArrayLikeAllClose(self, a, b, rtol=1e-6, atol=1e-6, msg=None):
                                                                      b.shape)
     self.assertEqual(a.shape, b.shape, shape_mismatch_msg)
 
+    msgs = [msg]
     if not np.allclose(a, b, rtol=rtol, atol=atol):
-      # Prints more details than np.testing.assert_allclose.
+      # Add more details than np.testing.assert_allclose.
       #
```
Comment (yifeif) [inline tensorflow/python/framework/test_util.py]: nit, here and below --> Adds more details to

**Review by yifeif — CHANGES_REQUESTED** [review]
Thanks @facaiy. Small nit.

**Comment by facaiy** [comment]
Thank you, yifei. I fixed it :-)

**Review by yifeif — APPROVED** [review]

**Pull request merged**